src/main/resources/META-INF/rewrite/easymock.yml

```diff
@@ -0,0 +1,156 @@
+  - org.openrewrite.java.migrate.RemoveMethodInvocation:
+      methodPattern: org.easymock.IExpectationSetters anyTimes()
+  - org.openrewrite.java.ChangeMethodName:
+      methodPattern: org.easymock.IExpectationSetters andReturn(..)
+      newMethodName: thenReturn
+      ignoreDefinition: true
+  - org.openrewrite.java.ChangeMethodName:
+      methodPattern: org.easymock.IExpectationSetters andThrow(java.lang.Throwable)
+      newMethodName: thenThrow
+  - org.openrewrite.java.ChangeMethodName:
+      methodPattern: org.easymock.IExpectationSetters andAnswer(..)
+      newMethodName: thenAnswer
+      ignoreDefinition: true
+  - org.openrewrite.java.ChangeMethodName:
+      methodPattern: org.easymock.IExpectationSetters andStubReturn(..)
+      newMethodName: thenReturn
+      ignoreDefinition: true
+  - org.openrewrite.java.ChangeMethodName:
+      methodPattern: org.easymock.IExpectationSetters andStubThrow(java.lang.Throwable)
+      newMethodName: thenThrow
+  - org.openrewrite.java.ChangeMethodName:
+      methodPattern: org.easymock.IExpectationSetters andStubAnswer(..)
+      newMethodName: thenAnswer
+      ignoreDefinition: true
+  - org.openrewrite.java.ChangeMethodTargetToStatic:
+      methodPattern: org.easymock.IMocksControl mock(..)
+      fullyQualifiedTargetTypeName: org.mockito.Mockito
+  - org.openrewrite.java.ChangeMethodTargetToStatic:
+      methodPattern: org.easymock.IMocksControl createMock(..)
+      fullyQualifiedTargetTypeName: org.mockito.Mockito
+  - org.openrewrite.java.migrate.RemoveMethodInvocation:
+      methodPattern: org.easymock.IMocksControl replay(..)
+  - org.openrewrite.java.migrate.RemoveMethodInvocation:
+      methodPattern: org.easymock.IMocksControl verify(..)
+  - org.openrewrite.java.migrate.RemoveMethodInvocation:
+      methodPattern: org.easymock.IMocksControl verifyRecording(..)
+  - org.openrewrite.java.migrate.RemoveMethodInvocation:
+      methodPattern: org.easymock.IMocksControl verifyUnexpectedCalls(..)
+  - org.openrewrite.java.migrate.RemoveMethodInvocation:
+      methodPattern: org.easymock.EasyMockSupport replayAll()
+  - org.openrewrite.java.migrate.RemoveMethodInvocation:
+      methodPattern: org.easymock.EasyMockSupport verifyAll()
+  - org.openrewrite.java.ChangeType:
+      oldFullyQualifiedTypeName: org.easymock.IArgumentMatcher
+      newFullyQualifiedTypeName: org.mockito.ArgumentMatcher
+  - org.openrewrite.java.ChangeMethodName:
+      methodPattern: org.easymock.EasyMock anyObject(..)
+      newMethodName: any
+  - org.openrewrite.java.ChangeMethodName:
+      methodPattern: org.easymock.EasyMock capture(..)
+      newMethodName: argThat
+  - org.openrewrite.java.ChangeMethodName:
+      methodPattern: org.easymock.EasyMock reportMatcher(..)
+      newMethodName: argThat
+  - org.openrewrite.java.ChangeMethodName:
+      methodPattern: org.easymock.EasyMock expect(..)
+      newMethodName: when
+  - org.openrewrite.java.migrate.RemoveMethodInvocation:
```

Comment:
```suggestion
  - org.openrewrite.java.RemoveMethodInvocations:
```

Reply:
Hello,

The recipe has remove from rewrite-migrate-java and the recipe is not same.
The recipe from core seem don't accept same methodPattern describe on [documentation](https://docs.openrewrite.org/reference/method-patterns).

A fix will coming ?

https://github.com/openrewrite/rewrite-migrate-java/pull/606/files#diff-fd5dcc89b3968363343fda3deced98bdc067a397c49471fc690849052a32fca1

Reply:
hi! Could you give an example of what you were able to remove with the old recipe that you're no longer able to remove now? Just trying to understand better. Perhaps an issue logged against openrewrite/rewrite, or draft PR with failing test would be best as this PR is about something different entirely.

https://github.com/openrewrite/rewrite/blob/cf71b0d927f09d89ecebc5c49c3fce00917e7ef3/rewrite-java/src/main/java/org/openrewrite/java/RemoveMethodInvocationsVisitor.java#L36

https://github.com/openrewrite/rewrite-static-analysis/blob/dac73436b8e929018383e9e19777131b6dec6b68/src/main/java/org/openrewrite/staticanalysis/RemoveMethodCallVisitor.java#L36

Reply:
hi! 

I created a PR on openrewrite/rewrite to missing usage : https://github.com/openrewrite/rewrite/pull/4754 

I re-read the comment with jevanlingen and understand when moving receipe from migrate-java to rewrite-java we lost the remove method if is static.

to be honest, I don't see a solution resolve the breaking change and compare with the visitor defined on rewrite-static-analysis.